docs/conversion_examples_gallery/recording_and_sorting/neuroscope.rst

```diff
@@ -0,0 +1,57 @@
+Neuroscope data conversion
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Example of how to convert ``Neuroscope`` to nwb:
+
+    >>> from datetime import datetime
+    >>> from dateutil import tz
+    >>> from pathlib import Path
+    >>> from nwb_conversion_tools import NeuroscopeRecordingInterface
+
+Then we creates a datetime object representing the date of the first of January of 2020 in the US/Pacific time-zone. 
+We use this for the metadata:
+    
+    >>> # Creates a datetime object with date the first of Junuary of 2020 in the US/Pacific time-zone
+    >>> metadata = interface.get_metadata()
+    >>> session_start_time = datetime(2020, 1, 1, 12, 30, 0, tzinfo=tz.gettz("US/Pacific"))
+    >>> metadata["NWBFile"] = dict(session_start_time=session_start_time)
+
+Finally, we indicate the location of the ``dat`` file and run the conversion.
+
+    >>> file_path = "./ephy_testing_data/neuroscope/test1/test1.dat"  
+    >>> interface = NeuroscopeRecordingInterface(file_path=file_path)
+    >>> save_path = "./nwb_neurocope_file.nwb"
+    >>> interface.run_conversion(save_path=save_path, metadata=metadata)
+
+Note that the file path is design to run in our CI. You have to make the file_path point to your local data file.
+
+If everything went right, this should return True
+
+    >>> Path(save_path).is_file()
+    True
+
+The other type of display:
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. code-block:: python
+
+    >>> from datetime import datetime
+    >>> from dateutil import tz
+    >>> from pathlib import Path
+    >>> from nwb_conversion_tools import NeuroscopeRecordingInterface
+    >>> 
+    >>> # For Neuroscope we need to pass the location of the `.dat` file
+    >>> file_path = "/home/heberto/ephy_testing_data/neuroscope/test1/test1.dat"
+    >>> interface = NeuroscopeRecordingInterface(file_path=file_path)
+    >>> 
+    >>> # Creates a datetime object with date the first of Junuary of 2020 in the US/Pacific time-zone
+    >>> metadata = interface.get_metadata()
+    >>> session_start_time = datetime(2020, 1, 1, 12, 30, 0, tzinfo=tz.gettz("US/Pacific"))
+    >>> metadata["NWBFile"] = dict(session_start_time=session_start_time)
```

Comment:
```suggestion
    >>> metadata["NWBFile"].update(session_start_time=session_start_time)
```

I prefer this syntax. The way you have it here, any fields in NWBFile other than `session_start_time` would be removed. You could also do `metadata["NWBFile"]["session_start_time"] = session_start_time`, but I prefer doing the dictionary update because you can assign multiple fields at once if you want to, e.g.

```python
metadata["NWBFile"].update(session_start_time=session_start_time, notes="hello")
```

Reply:
Right, this makes more sense. 
Done in the latest commit.

Reply:
@bendichter 
I just realized though, that this fails because the metadata does not necessarily have the key `NWBFile` at that point in time.